content/en/security/application_security/api-inventory/_index.md

```diff
@@ -10,18 +10,21 @@ further_reading:
 
 [API Security Inventory][7] monitors your API traffic to provide visibility into the security posture of your APIs, including:
 
-- **Authentication**: Type of authentication used, such as Basic Auth and API key.
+- **Authentication**: Wherther the API enforces authentication.
+- **Authentication Method**: Type of authentication used, such as Basic Auth and API key.
 - **Public Exposure**: Whether the API is processing traffic from the internet.
-- **Production status**: If the API is running in a production environment.
 - **Sensitive data flows**: Sensitive data handled by the API and flows between APIs.
 - **Attack Exposure**: If the endpoint is targeted by attacks (powered by [Application Threat Management][2]).
-- **Vulnerabilities**: If the endpoint contains a vulnerability (powered by [Software Composition Analysis][3]).
+- **Business Logic**: Business logic associated and business logic suggestions for this API.
```

Comment:
```suggestion
- **Business Logic**: Business logic and suggestions for improvement associated with this API.
```

Reply:
I don't think this suggestion matches my intent. But I guess that also shows that my initial sentence wasn't good enough

Reply:
Ok :) It is a little confusing! How about "Business logic and associated business logic suggestions for this API." ?

Reply:
Sure ! I think we need to add some Business Logic documentation page, as it seems it is missing. Should our team add it by ourselves, or is there some other process to follow?

Reply:
Thanks! Answered in slack